app/database/migrations/20191210114111-create-channelconfig.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('ChannelConfig', {
+      channel_config_id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      communication_channel_id: {
+        type: Sequelize.STRING,
+        allowNull: true
+      },
+      config: {
+        allowNull: false,
+        type: Sequelize.JSON
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      }
+    });
```

Comment:
onDelete: 'CASCADE', is missing to map with CommunicationChannels channel. please have a look other existing table how master and child tables are linked